swagger/src/main/scala/org/http4s/rho/swagger/Reflector.scala

```diff
@@ -0,0 +1,28 @@
+package org.http4s.rho.swagger
+
+import java.sql.Timestamp
+import java.util.Date
+
+import org.http4s.rho.bits.ResponseGenerator.EmptyRe
+
+object Reflector {
+  import scala.reflect.runtime.universe._
+
+  private[swagger] val primitives = {
+    Set[Type](typeOf[String], typeOf[Int], typeOf[Long], typeOf[Double],
```

Comment:
So  just for the sake of performance, couldn't you make this an array? Set access and iteration is slow, and the fact it's a set is pretty much an implementation detail, especially considering that it's private so it doesn't have to matter. See [haoyi's post](http://www.lihaoyi.com/post/BenchmarkingScalaCollections.html#sets-and-maps-are-slow)

That aside, you can also make it more performant by removing the need of concatenation with the empty set as a default as well. That can be a different call for the cases where that isn't used (which seems like most of them)

I'm nitpicking here btw. Premature optimization, etc, so feel free to ignore.

Reply:
This was already like this in 'rho', it just appears as a change because I moved it, though I don't fully remember why. I noted your commented anyway and will gladly have a new PR addressing it after this one is done.

Reply:
Generally speaking, I didn't care much about this code as its run once on startup, so as long as its not mega slow, it doesn't really matter. That said, an array is probably better.